### ISSUE TITLE
vscode extension

### ISSUE THREAD

**Issue by nicosuave** [uptech/alt#23]
I started building an Atom package a while back, but have since switched to VS Code as my primary editor (for non-JVM projects).

I'm volunteering myself to write a VS Code extension ✋

**Comment by nicosuave** [comment]
I'll begin this project as a separate repo, but tracking here for now

**Comment by drewdeponte** [comment]
@nicosuave yeah, we should definitely make it a separate repo. But, should just be a wrapper around this command.

**Comment by nicosuave** [comment]
Work has been busier than usual lately, but I should have a chance to at least start this weekend
